doc/yaml.md

```diff
@@ -70,7 +70,6 @@ YAML adds just a few things to this basic data model:
   * YAMLScript's "data mode" means the node is plain YAML data and "code mode"
     means it is meant to represent code expressions.
 
-
```

Comment:
is the following wording is correct? "YAMLScript has a data mode and a code-mode. YAMLScript data-mode is what colloquially is known as YAML syntax. The YAMLScript code-mode is a strict subset of YAML syntax."

Reply:
I wouldn't write it that way. I'll just rewrite this with links to `modes.md`.